openml_OS/views/pages/frontend/new/post.php

```diff
@@ -222,7 +222,7 @@
   $name = $this->input->post( 'study_title' );
   $alias = $this->input->post( 'study_alias' );
   $description = $this->input->post( 'description' );
-  $sid = $this->Study->create( $name, $alias, $description, $user_id );
+  $sid = $this->Study->legacy_create( $name, $alias, $description, $user_id );
```

Comment:
Should we take this down from the frontend? Or put a notice that they will be discontinued in their current form?

Reply:
Yes, take it down from the frontend

Reply:
take it down, would be great